### DISCUSSION TITLE
Can't apply for student developer packs

### DISCUSSION THREAD

**Discussion by ghost** [community/community#135883] [GitHub Education]
### Hi everyone,

I transferred to another school recently and just got my student ID. Application always rejected because my student ID doesn't have any enrollment date but I got a slip which has enrollment date also rejected. What should I do?

**Comment by emansarahafi** [comment]
Hi, you do not have to use your student ID only, you can use any other documentation from the university that has a date on it, such as an enrollment certificate, transcripts, etc.

**Reply by ghost** [reply]
I did but still declined by GitHub

**Reply by emansarahafi** [reply]
You're still eligible even if you're in high school, which document did you use? And what did the system say when it rejected your application? Also, did you use your school email address when applying for the GitHub Education Pack?
